src/Storages/System/attachSystemTables.cpp

```diff
@@ -96,105 +96,105 @@ namespace DB
-    attach<StorageSystemTableEngines>(context, system_database, "table_engines");
-    attach<StorageSystemContributors>(context, system_database, "contributors");
-    attach<StorageSystemUsers>(context, system_database, "users");
-    attach<StorageSystemRoles>(context, system_database, "roles");
-    attach<StorageSystemGrants>(context, system_database, "grants");
-    attach<StorageSystemRoleGrants>(context, system_database, "role_grants");
-    attach<StorageSystemCurrentRoles>(context, system_database, "current_roles");
-    attach<StorageSystemEnabledRoles>(context, system_database, "enabled_roles");
-    attach<StorageSystemSettingsProfiles>(context, system_database, "settings_profiles");
-    attach<StorageSystemSettingsProfileElements>(context, system_database, "settings_profile_elements");
-    attach<StorageSystemRowPolicies>(context, system_database, "row_policies");
-    attach<StorageSystemQuotas>(context, system_database, "quotas");
-    attach<StorageSystemQuotaLimits>(context, system_database, "quota_limits");
-    attach<StorageSystemQuotaUsage>(context, system_database, "quota_usage");
-    attach<StorageSystemQuotasUsage>(context, system_database, "quotas_usage");
-    attach<StorageSystemUserDirectories>(context, system_database, "user_directories");
-    attach<StorageSystemPrivileges>(context, system_database, "privileges");
-    attach<StorageSystemErrors>(context, system_database, "errors");
-    attach<StorageSystemWarnings>(context, system_database, "warnings");
-    attach<StorageSystemDataSkippingIndices>(context, system_database, "data_skipping_indices");
-    attach<StorageSystemLicenses>(context, system_database, "licenses");
-    attach<StorageSystemTimeZones>(context, system_database, "time_zones");
-    attach<StorageSystemBackups>(context, system_database, "backups");
-    attach<StorageSystemSchemaInferenceCache>(context, system_database, "schema_inference_cache");
-    attach<StorageSystemDroppedTables>(context, system_database, "dropped_tables");
+    attach<StorageSystemOne>(context, system_database, "one", "Used when the table is not specified explicitly, for example in queries like `SELECT 1`. Analog of the DUAL table in Oracle and MySQL.");
+    attach<StorageSystemNumbers>(context, system_database, "numbers", "Generates all natural numbers, starting from 0 (to 2^64 - 1, and then again) in sorted order.", false);
+    attach<StorageSystemNumbers>(context, system_database, "numbers_mt", "Multithreaded version of `system.numbers`. The result could be out of order.", true);
+    attach<StorageSystemZeros>(context, system_database, "zeros", "Generated unlimited amount of non-materialized zeros.", false);
+    attach<StorageSystemZeros>(context, system_database, "zeros_mt", "Multithreaded version of system.zeros.", true);
+    attach<StorageSystemDatabases>(context, system_database, "databases", "Contains all the information about all the databases within the current server.");
```

Comment:
"all the information" sounds a bit weird. Maybe just "Lists all databases/tab/es/columns of the current server." (l. 104/105/106)